tutorecommerce/templates/ecommerce/build/ecommerce/Dockerfile

```diff
@@ -46,6 +46,10 @@ RUN cd /openedx/requirements/ \
 {% for extra_requirement in ECOMMERCE_EXTRA_PIP_REQUIREMENTS %}RUN pip install {{ extra_requirement }}
 {% endfor %}
 
+{%- if patch("ecommerce-dockerfile-pre-assets") %}
```

Comment:
No need for this `if` condition; just apply the patch.

Reply:
Done!